doc/introduction/index.rst

```diff
@@ -272,6 +272,10 @@ can define your own configuration scenarios.
   are automatically launched at boot time by the hypervisor.  The User VMs can
   be Real-Time VMs (RTVMs), Safety VMs, or standard User VMs.
 
+  When selecting this scenario, ensure that ``reboot=acpi`` is included in the
```

Comment:
The document here is for ACRN High-Level Architecture. The scenario doesn't care the RTVM OS is Linux or others. But your patch specifies the "reboot=acpi" cmdline which works for Linux only.